scripts/generic-ci-tests.sh

```diff
@@ -99,19 +99,19 @@ case "$TEST_SUITE" in
     "lms-unit")
         case "$SHARD" in
             "all")
-                paver test_system -s lms --extra_args="--with-flaky" --cov_args="-p"
+                paver test_system -s lms --extra_args="--with-flaky --processes=-1" --cov_args="-p"
```

Comment:
I take it that this is intended to run everything through the multiprocessing pipeline, but to still run them single-threaded by default?